modules/assim.batch/R/pda.load.data.R

```diff
@@ -38,13 +40,19 @@ load.pda.data <- function(settings, con) {
     
     vars.used.index <- which(format$vars$bety_name %in% c(inputs[[i]]$variable.name))
     
+    if(format$file_name == "AmeriFlux.level2.h.nc"){
+      time.row <- NULL
+    }else{
+      time.row <- which(format$vars$bety_name %in% c("year", "month", "week", "day", "hour", "minute", "second"))
+    }
```

Comment:
It seems possible, @bcow is currently using `time.row` while calling `load.data` only in her workflow, `format` is also being passed to `load.data`, so` time.row` can be extracted from `format` within `load.data` function

@bcow would you have any objection moving it?